sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerLanguage.cs

```diff
@@ -3,10 +3,10 @@
 
 using Azure.Core;
 
-namespace Azure.AI.FormRecognizer.Models
+namespace Azure.AI.FormRecognizer
```

Comment:
Decided to leave in main namespace as this is an input object to the methods in client. Follows the same as `FormContentType`